build.sh

```diff
@@ -432,6 +441,14 @@ while getopts ":ch" options; do
             rm -rf "${BUILD_OUTPUT_DIR}"
         fi
         echo "Cleaned up '${BUILD_OUTPUT_DIR}'."
+        if [ -d "${BB_DIR}" ] && [ -z "${BB_DIR##*busybox-*}" ]; then
+            rm -rf "${BB_DIR}"
+        fi
```

Comment:
Is it needed to check if the DIR exists if you are gonna remove it?? I would just remove the if.

Reply:
Well, that is where all shit could hit the fan.. what if BB_DIR contained "/" ??

Reply:
Agreed, but for that you need to edit the script and put a " / " into BB_DIR. You will learn before committing to pay attention when typing ;-) Anyway, I just wonder if is it better to complicate things just to add some safe measures that is improbable to happen.